crate2nix/src/lib.rs

```diff
@@ -224,4 +224,5 @@ pub struct GenerateConfig {
     pub crate_hashes_json: PathBuf,
     pub nixpkgs_path: String,
     pub other_metadata_options: Vec<String>,
+    pub jobs: usize,
```

Comment:
I think that we should make the name to include "prefetch", i.e. "parallel_prefetches" because, for me, jobs is misleeding.